### PR TITLE
Add Record Output keyword for logging and match types for testing empty and not empty.

### PR DESCRIPTION
**Pull request by mharista**

### CODE REVIEW

File: AristaLibrary/Expect.py [hunk 1]
```diff
@@ -392,6 +392,48 @@ def refresh_command_output_on_devices(self, cmd=None):
...
+        # Use current switch if switch_id is not specified.
+        if switch_id:
+            switch = self.arista_lib.get_switch(switch_id)
+        else:
+            switch = self.arista_lib.get_switch()
+
+        result = None
+        if cmd:
+            if encoding not in ['text', 'json']:
+                encoding = 'text'
```
Comment (jerearista) [inline AristaLibrary/Expect.py]: I like where you went with a sane default but should we actually return an error if an invalid value was passed?

Comment (mharista) [reply AristaLibrary/Expect.py]: My thought is that a user will most of the time want text even if they typo it and won't necessarily think about what the valid options are for encoding. I also prefer to avoid failing a test case because of a logging error.

Comment (jerearista) [reply AristaLibrary/Expect.py]: Thanks.   Ok.
